networkx/algorithms/minors.py

```diff
@@ -445,37 +445,36 @@ def contracted_nodes(G, u, v, self_loops=True, copy=True):
     not be the same as the edge keys for the old edges. This is
     natural because edge keys are unique only within each pair of nodes.
 
+    This function is also available as `identified_nodes`.
```

Comment:
Should this be part of the "see also section"?  
I don't know if there is a good standard for showing synonnyms in docstrings

Reply:
It could - I don't think there are many advantages/disadvantages between the two. Either way a link to `identified_nodes` will appear in the docstring for `identified_nodes`. I don't know of any way to automagically change the links to the other alias for each docstring (e.g. `contracted_nodes` points to `identified_nodes` *and* vice-versa), but IMO it's not worth worrying about.

My main goal was to get rid of the duplicated Notes section, which violates the numpydoc standard.

Reply:
Ack!  I completely missed the fact that there were two "Notes" sections.

Looking at gnp_random_graph and its aliases binomial_graph and erdos_renyi_graph suggest that this should indeed go where you put it.  In the Notes section.  That doc_string even has a test of the alias:

```
    :func:`binomial_graph` and :func:`erdos_renyi_graph` are
    aliases for :func:`gnp_random_graph`.

    >>> nx.binomial_graph is nx.gnp_random_graph
    True
    >>> nx.erdos_renyi_graph is nx.gnp_random_graph
    True
```

That's probably more than we need...  Sorry for the noise.

Reply:
No worries! I reverted back to the version that has it in `Notes` instead of `See Also`. Thinking about it more, there may be a way to handle the aliases in the docstring using custom sphinx templates, but I'm not touching that in this PR :)